lib/api/api.go

```diff
@@ -1173,6 +1174,21 @@ func (s *service) getDBIgnores(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+func (s *service) getIgnoresRaw(w http.ResponseWriter, r *http.Request) {
+	qs := r.URL.Query()
+
+	folderPath := qs.Get("raw-ignore")
```

Comment:
```suggestion
	folderPath := qs.Get("path")
```

Again, lets call this `path`, please.